examples/player/simid_player.js

```diff
@@ -431,7 +430,7 @@ class SimidPlayer {
     this.videoTrackingEvents_.set("durationchange", () => {
       this.simidProtocol.sendMessage(MediaMessage.DURATION_CHANGED);
     });
-    this.videoTrackingEvents_.set("ended", this.videoComplete.bind(this));
+    // this.videoTrackingEvents_.set("ended", this.videoComplete.bind(this));
```

Comment:
don't we need to keep this callback?

Reply:
we no longer have a videoComplete function

Reply:
So if the creative does't call requestChangeDuration, how does the creative know when to terminate?

Reply:
How do we know when to terminate if there is no new duration?

Reply:
it does call requestChangeAdDuration and that function does exist